@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -638,6 +631,51 @@ export class VisibleTileSet {
         });
     }
 
+    private getTileImpl(
+        dataSource: DataSource,
+        tileKey: TileKey,
+        offset: number,
+        cacheOnly: boolean
+    ): Tile | undefined {
+        function updateTile(tileToUpdate?: Tile) {
+            if (tileToUpdate === undefined) {
+                return;
+            }
+            // Keep the tile from being removed from the cache.
+            tileToUpdate.frameNumLastRequested = dataSource.mapView.frameNumber;
+        }
+
+        if (!dataSource.cacheable) {
+            const resultTile = dataSource.getTile(tileKey);
```

Comment:
I think we should not call `dataSource.getTile(tileKey)` if `cacheOnly===true`

Reply:
done